WebDriverAgentLib/Categories/XCUIElement+FBIsVisible.m

```diff
@@ -28,7 +28,11 @@ @implementation XCElementSnapshot (FBIsVisible)
 
 - (BOOL)fb_isVisible
 {
-  return [(NSNumber *)[self fb_attributeValue:FB_XCAXAIsVisibleAttribute] boolValue];
+  // To avoid known issue "Error copying attributes -25202" in XCTest framework
+  if (self.frame.size.height > 0 && self.frame.size.width > 0) {
```

Comment:
I know it is aesthetic stuff, but please revert the check please, to make it look as fast return. And more comment here (eg. what you stated in diff description)